### PR TITLE
Puting all images and links in type carousel

### PR DESCRIPTION
**Pull request by julianecasier**

### CODE REVIEW

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Not seen by the DM

**Review by Dash293 — DISMISSED** [review]

File: frontend/src/components/TypeCarroussel/index.jsx [hunk 1]
```diff
@@ -32,6 +40,46 @@ export default function TypeCarroussel() {
             <img src={crime.picture} alt={crime.alt} />
           </Link>
         </SwiperSlide>
+        <SwiperSlide>
```
Comment (LuigiMenez) [inline frontend/src/components/TypeCarroussel/index.jsx]: use .map method

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Use .map method

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**